File: Swift-C++-IR-Comparison/test/collectionTypes/Swift/Arrays/MutableArray.swift
@main
struct MyMain {

  static func main() {
    var emptyArrayNoType: [Any] = []
    var emptyArray: [Int] = []

    var numberArrayNoType = [1, 2, 3, 4, 5]
    var numberArrayExplicitType: [Int] = [1, 2, 3, 4, 5]

    var stringArrayNoType = ["Test", "Input"]
    var stringArrayExplicitType: [String] = ["Test", "Input"]
  }
}

File: AutoTranspiledTests/Swift-to-Cpp/try-1/MutableArray.cpp

 #include <iostream>
 #include <string>
 #include <array>

 int main() {
	std::array<std::any,0> emptyArrayNoType;
	std::array<int,0> emptyArray;
	
	std::array<int, 5> numberArray = {1, 2, 3, 4, 5}; 
	std::array<std::string,2> stringArray = {"Test", "Input"};
	
	 return 0;
 }